weblate/static/style-bootstrap.css

```diff
@@ -225,7 +225,8 @@ textarea#id_comment {
   height: 4em;
 }
 div.CodeMirror.cm-s-weblate {
-  height: 5em;
+  min-height: 5em;
```

Comment:
The min-height seems to be source of displaying horizontal scroll bar as described in previous comments. 

```suggestion
```